src/Nancy/Culture/DefaultCultureService.cs

```diff
@@ -10,14 +10,17 @@
     public class DefaultCultureService : ICultureService
     {
         private readonly CultureConventions cultureConventions;
+        private readonly GlobalizationConfiguration configuration;
 
         /// <summary>
         /// Creates a new instance of DefaultCultureService
         /// </summary>
         /// <param name="cultureConventions">CultureConventions to use for determining culture</param>
-        public DefaultCultureService(CultureConventions cultureConventions)
+        /// <param name="configuration">GlobalizationConfiguration containing allowed cultures</param>
+        public DefaultCultureService(CultureConventions cultureConventions, GlobalizationConfiguration configuration)

```

Comment:
How can the service take a dependency on `GlobalizationConfiguration` ? That type isn't registered in the container in a reliable way. It would work for TinyIoC because of the auto-registration but for everything else it would fail. It would be better to take the dependency on `INancyEnvironment` and pull it out instead